message.go

```diff
@@ -238,6 +238,14 @@ func parsePrivateMessage(message *ircMessage) Message {
 
 	privateMessage.Emotes = parseEmotes(message.Tags["emotes"], privateMessage.Message)
 
+	firstMessage, ok := message.Tags["first-msg"]
+	if ok {
+		if firstMessage == "0" {
```

Comment:
why not `privateMessage.FirstMessage = firstMessage == "1"` seems more readable to me and less code

Reply:
Good call! Will fix.